tests/helpers.ts

```diff
@@ -1,55 +1,60 @@
+/**
+ * @license
+ * Copyright 2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 import * as nock from 'nock';
 import * as path from 'path';
 import * as balenaCLI from '../build/app';
 
 export const runCommand = async (cmd: string) => {
 	const preArgs = [process.argv[0], path.join(process.cwd(), 'bin', 'balena')];
 
-	const oldStdOut = process.stdout.write;
-	const oldStdErr = process.stderr.write;
-
 	const err: string[] = [];
 	const out: string[] = [];
 
-	// @ts-ignore
-	process.stdout.write = (log: string) => {
+	const stdoutHook = (log: string | Buffer) => {
 		// Skip over debug messages
-		if (!log.startsWith('[debug]')) {
+		if (typeof log === 'string' && !log.startsWith('[debug]')) {
 			out.push(log);
 		}
-		oldStdOut(log);
```

Comment:
The intention of the original author was to keep printing stdout and stderr to the screen in addition to capturing it, but it was not working (the output was not being printed to the screen). I've used a 4-year-old npm package (`intercept-stdout`) instead of rewriting it myself.